sklearn/cluster/_optics.py

```diff
@@ -34,10 +34,10 @@ class OPTICS(ClusterMixin, BaseEstimator):
     """Estimate clustering structure from vector array.
 
     OPTICS (Ordering Points To Identify the Clustering Structure), closely
-    related to DBSCAN, finds core sample of high density and expands clusters
-    from them [1]_. Unlike DBSCAN, keeps cluster hierarchy for a variable
+    related to DBSCAN, finds core samples of high density and expands clusters
+    from them [1]_. Unlike DBSCAN, it keeps cluster hierarchy for a variable
     neighborhood radius. Better suited for usage on large datasets than the
-    current sklearn implementation of DBSCAN.
+    current scikit-learn implementation of DBSCAN.
 
     Clusters are then extracted using a DBSCAN-like method
```

Comment:
The "then" makes it seem like we should mention the order list e.g.,
"Clusters are then extracted from the order list using...", though not sure what the right term is here

Reply:
going into that level of details will be hard, and it'll make this docstring very long, I'm not sure if we should do that here.

Reply:
What about just removing the "then" then?

"Clusters are extracted using a DBSCAN-like method ... "

Reply:
Though:

"Clusters are then extracted order list using a DBSCAN-like method"

isn't that much longer. Or are you saying that if we use the term "order list" we need to explain it further?

Reply:
for me "then" here implies "as the next step", but sure, you're the native speaker here 😁

Reply:
No I agree that "then" means "as the next step", I was trying to add the info of "next step after what"?

I know above we mention "Unlike DBSCAN, it keeps cluster hierarchy for a variable neighborhood radius", but then we talk about some other stuff, and it is not obvious what step happens *before* cluster extraction? i.e., what is the "then" following?

Reply:
Yes, sorry I was meaning more that we could improve the documentation by including the info of what happens before. Or just removing the "then", so it's not confusing as it's not clear atm what the "before" is to the "then".

Reply:
Okay, I used the term 'cluster-order' as it seems to be the term used in the paper.

Reply:
maybe you forgot to push?

Reply:
Yes done now. Had to do something else in the middle.